text edits to holding, checkin  and projection

MrPyroMeiniac opened the pull request.

MrPyroMeiniac committed:
text edits to holding, checkin  and projection

kimberleynyam commented on app/holding/index.html:
```diff
@@ -25,8 +25,8 @@
-            <h4 class="card-title">Please confirm</h4>
-            <p class="card-text geolocation-message"> <span class="extra-message"></span>Please tap the button below when the projector gives you the sign that we need your input</p>
+            <h4 class="card-title">ATTENTION!</h4>
+            <p class="card-text geolocation-message"> <span class="extra-message"></span>Tap the button below when the projector gives you the sign that we need your input</p>
```
This all sounds really good, for line 29 how about "Tap the button below when the projector indicates that we need your input" or "signals that we need your input" - "gives you the sign that" sounds a little wordy

MrPyroMeiniac committed:
longi/latitude corrections

kimberleynyam commented on projections/index.html:
```diff
@@ -103,7 +103,7 @@
 
           <h1>Warning</h1>
-          <p>You don't have long with david</p>
+          <p>You don't have long left with david</p>
```
Probably sounding like a Year One teacher but is David without a capital intentional?

MrPyroMeiniac replied:
thanks Kim :smiley_cat:

MrPyroMeiniac committed:
Update index.html

MrPyroMeiniac committed:
Update index.html

MrPyroMeiniac committed:
Merge remote-tracking branch 'refs/remotes/origin/master' into Richard/text-edit

MrPyroMeiniac committed:
Merge branch 'Richard/text-edit' of https://github.com/bridgeatwaterloo/david into Richard/text-edit

aliblackwell commented on app/checkin/checkin.js:
```diff
@@ -17,16 +17,15 @@ function processCheckin() {
 
       // Our logic. If the distance between our user and the target is less than 100m, show the login screen, else update the view with a helpful message
-      if (distance < 1000) {
+      if (distance < 100) {
```
This is 100km still! 100metres would be 0.1 @MrPyroMeiniac

aliblackwell commented:
@MrPyroMeiniac could we also look at https://github.com/bridgeatwaterloo/david/issues/105 in this PR? We need to tell the user to enable location services if they get a geolocation error.

Remember: `navigator.geolocation.getCurrentPosition(successCallback,` errorCallback, options)`

We are using an anonymous function for the successCallback:

```
navigator.geolocation.getCurrentPosition(function(position) {
  // success in here
});
```

So we'd need to do something like this:

```
navigator.geolocation.getCurrentPosition(function(position) {
  // success in here
}, function(error) {
  // error in here
});
```

or refactor it like this:

```

function gotLocation(position) {
  // success in here
}

function gotError(error) {
  // error in here 
}

navigator.geolocation.getCurrentPosition(gotLocation, gotError);
```

MrPyroMeiniac committed:
stuff

fixed geolocation threshold and added error message if geolocation is
not available

MrPyroMeiniac committed:
quick change

quick change

MrPyroMeiniac committed:
Merge remote-tracking branch 'refs/remotes/origin/master' into Richard/text-edit

# Conflicts:
#	app/checkin/checkin.js

aliblackwell commented:
Well done @MrPyroMeiniac @francissmedley !
